path.go

```diff
@@ -500,11 +500,27 @@ func newSelectorNode(selector string) *selectorNode {
 }
 
 func (n *selectorNode) filter(node ast.Node) (ast.Node, error) {
+	selector := n.selector
+	if len(selector) > 0 && selector[0] == '\'' {
+		selector = selector[1 : len(selector)-1]
```

Comment:
Is there any need to check that single quote is not used as the terminating character as in `'selector` ?